resources/cluster-essentials/files/subscriptions.eventing.kyma-project.io.crd.yaml

```diff
@@ -0,0 +1,54 @@
+
```

Comment:
Now that the eventing-controller Beb integration is merged, let's use the updated CRD

Reply:
Done